.github/workflows/nightly-latest-packages-validation.yml

```diff
@@ -0,0 +1,36 @@
+# In order to be updated when nightlies fail, please subscribe for updates on PR:
+# ---> https://github.com/facebook/metro/pull/1314 <---
+# where comments will be published on fail.
+
+# This is a bit of a workaround tackling the lack of an organic way
+# to notify certain people when a Github workflow fails:
+# https://github.com/orgs/community/discussions/18039
+
+name: facebook/metro/nightly-latest-packages-validation
+on:
+  schedule:
+    # Everyday at at 5:00 UTC (22:00 USA West Coast, 06:00 London)
+    - cron:  '0 5 * * *'
+  push:
+
+jobs:
+  test:
+    uses: ./.github/workflows/test.yml
+    with:
+      recreate-yarn-lock: true
+
+  comment-on-pr-for-failures:
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: ${{ always() && needs.test.result == 'failure' }}
+    steps:
+      - uses: actions/github-script@v7
+        with:
+          script: |
+            // github.rest.issues.createComment({
+            //   // see https://github.com/facebook/metro/pull/1314
+            //   issue_number: 1314,
+            //   owner: context.repo.owner,
+            //   repo: context.repo.repo,
+            //   body: 'A nightly test failed in `${{ github.workflow }}` in [run ${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})!',
+            // });
```

Comment:
Are we intending to uncomment this code?

Reply:
I'll return this code once I've finished with the build code. otherwise it will spam us with failures